src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3119,7 +3119,7 @@ A catch() block after a catch (System.Exception e) block can catch non-CLS excep
     <value>A namespace declaration cannot have modifiers or attributes</value>
   </data>
   <data name="ERR_BadPlatformType" xml:space="preserve">
-    <value>Invalid option '{0}' for /platform; must be anycpu, x86, Itanium, arm, arm64 or x64</value>
+    <value>Invalid option '{0}' for /platform; must be anycpu, x86, Itanium, arm, arm64, loongarch64 or x64</value>
```

Comment:
The issue @jjonescz linked to describes the problem:
```
chenguohui@debian-230:~/target-runtime/hello$ ~/target-runtime/dotnet-sdk-9.0.100-linux-loongarch64/dotnet bin/Release/net9.0/linux-loongarch64/hello.dll
Unhandled exception. System.BadImageFormatException: Could not load file or assembly '/home/chenguohui/target-runtime/hello/bin/Release/net9.0/linux-loongarch64/hello.dll'. An attempt was made to load a program with an incorrect format.
```

@chenguohui What specifically is wrong with the binary? It's not obvious from the information provided that there is a problem with the compiler